### PR TITLE
OS-8548 Typo in altexec(8) man page

### PR DESCRIPTION
**Pull request by bahamat**
Change-Id: I548adb0203a335e097ef00315e3f5620405514a9

### CODE REVIEW

**Comment by bahamat** [comment]
Tested this with `man usr/src/man/man8/altexec.8` to view it directly.

File: usr/src/man/man8/altexec.8 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-n.\"
+.\"
```
Comment (danmcd) [inline usr/src/man/man8/altexec.8:1]: OH FOR CRYING OUT LOUD, stupid "I didn't press 'n'o in emacs, right?" bug.  I'm sorry.

**Review by danmcd — APPROVED** [review]

**Comment by danmcd** [comment]
> Tested this with `man usr/src/man/man8/altexec.8` to view it directly.

Put that into the bug report and I'll IA this.

**Comment by danmcd** [comment]
> > Tested this with `man usr/src/man/man8/altexec.8` to view it directly.
> 
> Put that into the bug report and I'll IA this.

Pfft, I'll do that myself.

**Pull request merged**